development/mimir-microservices-mode/config/prometheus.yaml

```diff
@@ -6,53 +6,84 @@ global:
 scrape_configs:
   - job_name: mimir-microservices-mode/distributor
     static_configs:
-      - targets: ['distributor-1:8000', 'distributor-2:8001']
+      - targets: ['distributor-1:8000']
         labels:
           cluster: 'docker-compose'
           namespace: 'mimir-microservices-mode'
+          pod: 'distributor-1'
+      - targets: ['distributor-2:8001']
+        labels:
+          cluster: 'docker-compose'
+          namespace: 'mimir-microservices-mode'
+          pod: 'distributor-2'
```

Comment:
Instead of adding `pod` to all targets, can we add relabeling rule to add `pod` from instance automatically? `instance` is `<host>:<port>` of target URL's that was scraped.

Reply:
We should also add similar rule (or static `pod`) to `grafana-agent.yaml` file, so that both Grafana Agent and Prometheus produce the same labels.